lib/rubocop/cop/layout/empty_lines_around_arguments.rb

```diff
@@ -45,7 +45,8 @@ class EmptyLinesAroundArguments < Base
         MSG = 'Empty line detected around arguments.'
 
         def on_send(node)
-          return if node.single_line? || node.arguments.empty?
+          return if node.single_line? || node.arguments.empty? ||
+                    node.receiver && node.receiver.loc.last_line != node.loc.selector.line
```

Comment:
I'd probably wrap this in a method, as it's a bit hard to understand as it is.

Reply:
Sure! I updated this PR.